libraries/plugins/peerplays_sidechain/sidechain_net_handler_bitcoin.cpp

```diff
@@ -703,6 +706,38 @@ bool bitcoin_rpc_client::walletpassphrase(const std::string &passphrase, uint32_
    return false;
 }
 
+uint64_t bitcoin_rpc_client::outputamount(const std::string &txid, uint32_t output_num)
+{
```

Comment:
Why do we have this one?

Reply:
I need exact input amount in satoshis for signing. This amount can't restored from double value we use in other places,  so I query it from bitcoin node